.github/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+### Description
+What does this do? What problem/s does it solve? What issue numbers does it
+resolve?
+
+### Checklist
+- [ ] I have read [CONTRIBUTING.md](https://github.com/sopel-irc/sopel/blob/master/CONTRIBUTING.md)
+- [ ] My patch is based on the correct branch (eg. `6.6.x` or `master`)
+- [ ] I am opening this PR against that branch
+- [ ] I can and do license this code under EFLv2
+- [ ] `checkstyle.sh` and `pytest_run.py` show no issues
```

Comment:
Reference `make qa` (and the sub-steps, `make quality` & `make test`) instead of the scripts themselves.

Reply:
Is it important to mention `quality`/`test` on their own, or just `qa`?

Reply:
It's just faster to check `quality` separately. `make test` takes quite a while, even on SSD machines.

Reply:
Only 30 seconds on my machine, and I think we'll get better compliance with a simpler instruction since we definitely want both run before a PR. If they want to save time while developing, the others are documented somewhere I assume?

Reply:
Not really, no. Only as targets in the Makefile if one reads it.

And `make quality` takes about 5 sec, so.

Reply:
Well, that should be fixed (and if we're using `make`, maybe the scripts put under `contrib/`?, but in the meantime I'll mention all three:

### Description
What does this do? What problem/s does it solve? What issue numbers does it
resolve, if any?

### Checklist
- [ ] I have read [CONTRIBUTING.md](https://github.com/sopel-irc/sopel/blob/master/CONTRIBUTING.md)
- [ ] I can and do license this contribution under the EFLv2
- [ ] There are no test or style issues shown by `make qa` (runs both `make
  quality` and `make test`)
- [ ] I have tested the functionality of the things this change touches